src/interface.ts

```diff
@@ -406,6 +411,8 @@ interface IStreamingAPI {
    *    On failure, the promise will be rejected with the entire API response.
    */
   request(stream: string, payload: object, requestId?: string): Promise<object>;
+
+  requestMultiple(stream: string, payload: object, callback:RequestMultipleHandler, requestId?: string): RequestMultipleCancel;
```

Comment:
nit: spacing
```suggestion
  requestMultiple(stream: string, payload: object, callback: RequestMultipleHandler, requestId?: string): RequestMultipleCancel;
```

Reply:
I'd like to see some JSDocs for the method before this gets released, but that can wait.

Reply:
Added some JSDocs for this in 635369f.